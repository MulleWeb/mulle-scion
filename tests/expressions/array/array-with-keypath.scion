{% 
   dict = @{ @"xxx", @"name" }
   deep = @( dict) 
 %}
{{ deep[ 0] }}
{{ deep[ 0].name }}
{{ deep[ 0].name.class | [self hasSuffix:@"String"] }}
