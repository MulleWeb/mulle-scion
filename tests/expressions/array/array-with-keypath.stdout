{
    name = xxx;
}
xxx
1
